### PR TITLE
Rename `assopy.User` to `assopy.AssopyUser` (models and factories)

### PR DESCRIPTION
**Pull request by umgelurgel**
* Also remove a duplicate `AssopyUserFactory`.

### CODE REVIEW

File: assopy/forms.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def clean_captcha_answer(self):
 
     def clean_email(self):
         email = self.cleaned_data['email']
-        if auth.models.User.objects.filter(email__iexact=email).count() > 0:
+        if auth.models.AssopyUser.objects.filter(email__iexact=email).count() > 0:
```
Comment (artcz) [inline assopy/forms.py]: that's not going to work, does it?

File: assopy/migrations/0007_auto_20180927_1118.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (artcz) [inline assopy/migrations/0007_auto_20180927_1118.py]: I'd replace that auto generated migration (file) name with something custom. I'll add that to the development guidelines as well.

File: pycon/dev-settings.py [hunk 3]
```diff
@@ -26,12 +26,15 @@
 
 TEMPLATES[0]['OPTIONS']['debug'] = True
 
-INSTALLED_APPS = INSTALLED_APPS + (
+# The cast to set to tuple is a workaround allowing to have `django_extensions`
```
Comment (artcz) [inline pycon/dev-settings.py]: not sure about this. especially about this `tuple()` maybe use `list()` instead? or just make sure that those apps are not duplicated?

File: assopy/forms.py (hunk 1 above)
Comment (umgelurgel) [reply assopy/forms.py]: Thanks for catching this, seems I got too eager renaming the model :)

File: pycon/dev-settings.py (hunk 3 above)
Comment (umgelurgel) [reply pycon/dev-settings.py]: `INSTALLED_APPS` is a `tuple` in `pycon.settings`, that's why I kept it here. I agree the change is controversial - I'll remove this change for now as it's something that's going to require discussion and I wouldn't want it to delay merging this PR.

File: assopy/migrations/0007_rename_assopy_user_model.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('assopy', '0006_add_bank_to_payment_options'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
```
Comment (artcz) [inline assopy/migrations/0007_rename_assopy_user_model.py]: does it also rename the db table?

Comment (umgelurgel) [reply assopy/migrations/0007_rename_assopy_user_model.py]: Had to double check, but I can confirm it does rename the DB table:

image

**Comment by artcz** [comment]
@umgelurgel python3 rebase needed. :)

**Comment by umgelurgel** [comment]
@artcz rebased and ready for review :)

File: conference/management/commands/create_initial_data_for_dev.py [hunk 5]
```diff
@@ -76,16 +79,28 @@ def handle(self, *args, **options):
...
             username='admin', email='admin@admin.com', password='europython')
 
         print("Creating regular users")
-        assopy_models.User.objects.create_user(
-            email='alice@europython.eu', password='europython', active=True, send_mail=False)
-        assopy_models.User.objects.create_user(
-            email='bob@europython.eu', password='europython', active=True, send_mail=False)
-        assopy_models.User.objects.create_user(
-            email='cesar@europython.eu', password='europython', active=True, send_mail=False)
+        AssopyUser.objects.create_user(
```
Comment (umgelurgel) [inline conference/management/commands/create_initial_data_for_dev.py:86]: Good catch! :+1:

**Review by artcz — APPROVED** [review]
LGTM

**Pull request merged**